src/components/MultiHopTrade/helpers.ts

```diff
@@ -36,7 +36,7 @@ export const getMixpanelEventData = () => {
   return {
     buyAsset: compositeBuyAsset,
     sellAsset: compositeSellAsset,
-    fiatAmount: sellAmountBeforeFeesFiat,
+    amountUSD: sellAmountBeforeFeesUserCurrency,
```

Comment:
revert, this is "user" facing and should be in a separate PR for granularity

Reply:
https://github.com/shapeshift/web/pull/4892/commits/95661a9fe9972bd5668a21250647afc37d1efb73